conan/tools/cmake/cmakedeps.py

```diff
@@ -54,17 +54,17 @@
                    conan_message(STATUS "Skipping already existing target: ${_LIB_NAME}")
                endif()
                list(APPEND _out_libraries_target ${_LIB_NAME})
-               conan_message(STATUS "Found: ${CONAN_FOUND_LIBRARY}")
+               conan_message(DEBUG "Found: ${CONAN_FOUND_LIBRARY}")
            else()
-               conan_message(ERROR "Library ${_LIBRARY_NAME} not found in package")
+               conan_message(FATAL_ERROR "Library '${_LIBRARY_NAME}' not found in package. If '${_LIBRARY_NAME}' is a system library, declare it with 'cpp_info.system_libs' property")
```

Comment:
Please confirm behavior, "ERROR" is not a valid tag, Do we want to fail, right? Old generators didn't fail but printed a message.